exercises/git-ignore/files/file2.txt

```diff
@@ -1 +0,0 @@
-Ignore this file, but the rule should not show up in project gitignore
```

Comment:
Do you know how to do this?

Reply:
Yeah I added it to the exclude file in .git